.gitbook/assets/ReadonlyREST-api.yml

```diff
@@ -194,6 +194,55 @@ paths:
               examples:
                 UnsupportedMediaTypeResponseExample:
                   $ref: "#/components/examples/UnsupportedMediaTypeResponseExample"
+
+  /api/ror/settings/all_tenants:
```

Comment:
WDYT about `GET /api/ror/user/tenants` instead?

Reply:
Yeah, sounds more rest and self-explanatory